ocaml/xapi/nm.ml

```diff
@@ -309,6 +309,20 @@ let determine_static_routes net_rc =
   else
     []
 
+let update_master_pif_mac ~__context bond_record (bond_master_record : API.pIF_t) bond_master =
```

Comment:
We often use the prefix `maybe_` for functions that may or may not do something depending on the input.